api/peering.go

```diff
@@ -0,0 +1,213 @@
+package api
+
+import (
+	"context"
+	"fmt"
+)
+
+// PeeringState enumerates all the states a peering can be in
+type PeeringState int32
+
+const (
+	// Undefined represents an unset value for PeeringState during
+	// writes.
+	UNDEFINED PeeringState = 0
+	// INITIAL Initial means a Peering has been initialized and is awaiting
+	// acknowledgement from a remote peer.
+	INITIAL PeeringState = 1
+	// Active means that the peering connection is active and healthy.
+	// ACTIVE PeeringState = 2
```

Comment:
Should this line be commented out?

Reply:
I was going to keep this in since there is a commented out line to use that state for better testing